### PR TITLE
feat: TE7-T1022: Implement Drawer active background for all screens

### PR DESCRIPTION
**Pull request by jigar-s-simform**
Implement Drawer active background for all screens

### CODE REVIEW

File: app/components/custom-drawer/CustomDrawer.tsx [hunk 1]
```diff
@@ -27,82 +37,120 @@ const CustomDrawer = (props: DrawerContentComponentProps): JSX.Element => {
...
+              ? {
+                  uri: userDetails?.avatar,
+                }
+              : userDetails.avatar
+          }
+          style={styles.profileImgStyles}
+        />
+        <View style={styles.nameEmailContainer}>
+          <Text style={styles.detailNameText}>
+            {`${userDetails?.first_name} ${userDetails?.last_name}`}
```
Comment (amitkajwani-simformsolutions) [inline app/components/custom-drawer/CustomDrawer.tsx]: Use proper optional chaining.

File: app/components/custom-drawer/CustomDrawer.tsx [hunk 2]
```diff
@@ -27,82 +37,120 @@ const CustomDrawer = (props: DrawerContentComponentProps): JSX.Element => {
...
-                color={Colors[theme]?.white}
-              />
-              <Text style={styles.detailText}>{userDetails?.email}</Text>
-            </View>
+      <View style={styles.detailsContainer}>
+        <Image
+          source={
+            typeof userDetails?.avatar !== 'number'
+              ? {
+                  uri: userDetails?.avatar,
```
Comment (amitkajwani-simformsolutions) [inline app/components/custom-drawer/CustomDrawer.tsx]: Use proper optional chaining.

File: app/components/custom-drawer/CustomDrawer.tsx [hunk 3]
```diff
@@ -27,82 +37,120 @@ const CustomDrawer = (props: DrawerContentComponentProps): JSX.Element => {
...
+          <Text style={styles.detailNameText}>
+            {`${userDetails?.first_name} ${userDetails?.last_name}`}
+          </Text>
+          <View style={styles.emailContainer}>
+            <EnvelopeSimple
+              size={moderateScale(25)}
+              weight="bold"
+              color={Colors[theme].white}
+            />
+            <Text style={styles.detailText}>{userDetails?.email}</Text>
```
Comment (amitkajwani-simformsolutions) [inline app/components/custom-drawer/CustomDrawer.tsx]: Use proper optional chaining.

File: app/navigation/home-tab/HomeTabButton.tsx [hunk 4]
```diff
@@ -27,6 +31,21 @@ const TabButton = (props: TabButtonProps): JSX.Element => {
...
   }, [focused]);
 
+  useFocusEffect(() => {
+    if (
+      props.to?.toLowerCase().includes(NavigationRoutes.HomeStack.toLowerCase())
+    ) {
+      dispatch(setRoute(NavigationRoutes.HomeScreen));
+    }
+    if (
+      props.to
```
Comment (amitkajwani-simformsolutions) [inline app/navigation/home-tab/HomeTabButton.tsx]: refractor useFocusEffect

File: app/navigation/home-tab/HomeTabs.tsx [hunk 5]
```diff
@@ -28,6 +32,10 @@ const HomeTabs: FC = () => {
               {...props}
               image={Icons.homeIcon}
               title={Strings.bottomNavigationTitles.home}
+              onPress={() => {
+                navigateWithParam(NavigationRoutes.HomeScreen)
+                dispatch(setRoute(NavigationRoutes.HomeScreen))
```
Comment (amitkajwani-simformsolutions) [inline app/navigation/home-tab/HomeTabs.tsx]: create a separate

File: app/navigation/home-tab/HomeTabs.tsx [hunk 6]
```diff
@@ -44,6 +52,10 @@ const HomeTabs: FC = () => {
               {...props}
               image={Icons.createUserIcon}
               title={Strings.bottomNavigationTitles.create}
+              onPress={() => {
+                navigateWithParam(NavigationRoutes.CreateUserScreen)
+                dispatch(setRoute(NavigationRoutes.CreateUserScreen))
```
Comment (amitkajwani-simformsolutions) [inline app/navigation/home-tab/HomeTabs.tsx]: create a separate

File: app/navigation/home-tab/HomeTabs.tsx [hunk 7]
```diff
@@ -58,6 +70,10 @@ const HomeTabs: FC = () => {
               {...props}
               image={Icons.videosIcon}
               title={Strings.bottomNavigationTitles.videos}
+              onPress={() => {
+                navigateWithParam(NavigationRoutes.VideoStack)
+                dispatch(setRoute(NavigationRoutes.VideoStack))
```
Comment (amitkajwani-simformsolutions) [inline app/navigation/home-tab/HomeTabs.tsx]: create a separate

File: app/navigation/home-tab/HomeTabs.tsx [hunk 8]
```diff
@@ -74,8 +90,13 @@ const HomeTabs: FC = () => {
               {...props}
               image={Icons.profileIcon}
               title={Strings.bottomNavigationTitles.profile}
+              onPress={() => {
+                navigateWithParam(NavigationRoutes.ProfileScreen)
+                dispatch(setRoute(NavigationRoutes.ProfileScreen))
```
Comment (amitkajwani-simformsolutions) [inline app/navigation/home-tab/HomeTabs.tsx]: create a separate

**Review by amitkajwani-simformsolutions — CHANGES_REQUESTED** [review]

File: app/components/custom-drawer/CustomDrawer.tsx (hunk 1 above)
Comment (jigar-s-simform) [reply app/components/custom-drawer/CustomDrawer.tsx]: where in styles?

Comment (amitkajwani-simformsolutions) [reply app/components/custom-drawer/CustomDrawer.tsx]: userDetails?.first_name

Comment (amitkajwani-simformsolutions) [reply app/components/custom-drawer/CustomDrawer.tsx]: Just check the line where I am putting a comment.

Comment (jigar-s-simform) [reply app/components/custom-drawer/CustomDrawer.tsx]: done already

**Review by amitkajwani-simformsolutions — APPROVED** [review]

**Pull request merged**